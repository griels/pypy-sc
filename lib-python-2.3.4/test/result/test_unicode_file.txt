command: /usr/bin/python /home/arigo/pypysrc/pypy/tool/alarm.py 900 /home/arigo/pypysrc/pypy/interpreter/py.py --oldstyle /home/arigo/pypysrc/lib-python-2.3.4/test/test_unicode_file.py
run by: arigo@tismerysoft.de
sys.platform: linux2
sys.version_info: (2, 3, 5, 'final', 0)
startdate: Wed Apr 27 18:29:36 2005
pypy-revision: 11518
OUTPUT TEST
see output in: /home/arigo/pypysrc/lib-python-2.3.4/test/result/test_unicode_file.out
============================================================
faking <type 'unicode'>
faking <type 'module'>
faking <type 'file'>
faking <type 'posix.stat_result'>
faking <type 'posix.statvfs_result'>
faking <type '_sre.SRE_Pattern'>
faking <type 'function'>
Traceback (application-level):
  File "/home/arigo/pypysrc/lib-python-2.3.4/test/test_unicode_file.py", line 13 in ?
    raise TestSkipped("No Unicode filesystem semantics on this platform.")
TestSkipped: No Unicode filesystem semantics on this platform.
========================== closed ==========================
execution time: 11.0879468918 seconds
exit status: 1
