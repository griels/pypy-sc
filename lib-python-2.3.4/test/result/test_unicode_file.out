test_unicode_file
